plugins/stack-overflow/src/api/StackOverflowClient.ts

```diff
@@ -46,6 +46,7 @@ export class StackOverflowClient implements StackOverflowApi {
   }): Promise<StackOverflowQuestion[]> {
     const params = qs.stringify(options.requestParams, {
       addQueryPrefix: true,
+      arrayFormat: 'repeat',
```

Comment:
would you be able to create a test for this use case in `plugins/stack-overflow/src/api/StackOverflowClient.test.ts` with the expected result?

Reply:
Reworked the test!